assets/src/edit-story/components/panels/design/preset/header.js

```diff
@@ -92,15 +64,18 @@ function PresetsHeader({
             isEditMode={isEditMode}
           >
             {isEditMode ? __('Done', 'web-stories') : <Icons.Pencil />}
```

Comment:
Just saw that the "Done" button shouldn't be like the other icon buttons -- will fix that.